README.md

```diff
@@ -185,3 +187,15 @@ Tip (hashing password for credentials):
 ```bash
 python3 -c "from werkzeug.security import generate_password_hash; print(generate_password_hash('password'))"
 ```
+
+## Test stand
+
+For creating test stand use docker-compose:
+
+```bash
+docker-compose up
+```
+
+It up minio ad s3 storage and creates rws docker image. Default
```

Comment:
`minio ad s3` -> `minio as S3`?

Reply:
fixed

Reply:
![image](https://github.com/tarantool/rws/assets/57662928/8dbc6e47-d688-4812-8f86-708327ca1d76)
 No, it isn't fixed.

Reply:
My bad) Fixed!